src/core/utils/deepMerge.ts

```
@@ -1,24 +1,27 @@
-        }
+    return objects.reduce(
+        (prev, obj) => {
+            if (!isObject(prev) || !isObject(obj)) {
+                return obj;
+            }
 
-        Object.keys(obj).forEach((key) => {
-            const pVal = prev[key as keyof {}] as object | any[];
-            const oVal = obj[key as keyof object] as object | any[];
+            Object.keys(obj).forEach((key) => {
+                const pVal = prev[key as keyof {}] as object | any[];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)